frontend/app/add-balance/ssl-success/page.tsx

```
@@ -0,0 +1,109 @@
+          <Link
+            href="/"
+            className="inline-flex items-center space-x-2 bg-white text-indigo-700 px-6 py-3 rounded-full font-medium shadow hover:bg-indigo-50 transition"
+          >
+            <span>Go to Homepage</span>
+            <svg
+              className="w-5 h-5"
+              fill="none"
+              stroke="currentColor"
+              viewBox="0 0 24 24"
+            >
+              <path
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)